assets/src/edit-story/components/panels/textStyle/font.js

```diff
@@ -64,21 +64,19 @@ const getFontWeights = ({ weights }) => {
 };
 
 function FontControls({ selectedElements, pushUpdate }) {
-  const fontFamily = getCommonValue(
-    selectedElements,
-    ({ font }) => font?.family
-  );
+  const font = getCommonValue(selectedElements, 'font');
```

Comment:
You can't, unfortunately, do `getCommonValue` on an object, since they are all different. What you can do is revert back to `fontFamily = getCommonValue` and then do `const font = useMemo(() => fontFamily != MULTIPLE && getFontByName(fontFamily), [...])` and then you can use that memoized object for all other needs, when it's available, of course.